src/styles/components/tile.scss

```diff
@@ -31,12 +35,32 @@ $tile: '.tile';
     margin: 0;
     line-height: 1.2;
   }
-  padding: $global-padding/2 0 0 0;
-  cursor: pointer;
 
   &__header {
+    @include fs-font-settings('H3');
     display: block;
+    margin-top: 0.5em;
```

Comment:
should this be `rem`?

Reply:
You tell me, I'm not attached to anything. rem will be fixed, em will give us more or less space depending on the font-size of the title (in case it's overriden)

Reply:
Actually, not needed now using the new helper classes. I think the spacing is now correct